soapui/src/main/java/com/eviware/soapui/support/ClassUtils.java

```diff
@@ -27,7 +27,23 @@
         return result;
     }
 
-    private static void addImplementedAndExtendedClasses(Class<?> clazz, ArrayList<Class<?>> result) {
+    public static List<Class<?>> getSuperInterfaces(Class clazz) {
+        ArrayList<Class<?>> result = new ArrayList<Class<?>>();
+        addImplementedSuperInterfaces(clazz, result);
+        return result;
+    }
+    
+    private static void addImplementedSuperInterfaces(Class clazz, ArrayList<Class<?>> result) {

```

Comment:
Again, the name is rather confusing. I suggest addImplementedInterfacesFromSuperClass() or something like that.